packages/components/src/lib/hooks/useSquiggle.ts

```diff
@@ -100,6 +101,13 @@ export function useSquiggle(args: SquiggleArgs): UseSquiggleOutput {
         project.setContinues(sourceId, continues);
         await project.run(sourceId);
         const output = project.getOutput(sourceId);
+
+        //Set the output to the window so that it can be accessed by users/developers there
+        //This is useful for debugging
+        if (window) {
+          window[WINDOW_VARIABLE_NAME] = output;
```

Comment:
How about this:
We do (1), and also, we set each playground to have it's own ID, that we use for it. 
Like, window.squiggleOutput.[Playground_ID]

We can generate the ID with https://react.dev/reference/react/useId.

If we do that, we could also save the ``project`` and ``code``, and anything else, if we want.

Reply:
Yes, that's also an option.

OTOH, I've started to have doubts about the usefulness of this feature, compared to using built-in React dev tools:
<img width="282" alt="Captura de pantalla 2023-11-20 a la(s) 15 39 09" src="https://github.com/quantified-uncertainty/squiggle/assets/89368/a7dc212f-9d00-4f67-af07-149af81b045d">

I just tried it and it works; it's slightly hard to locate SquiggleViewer component because it doesn't have a name (I'm not sure why; we do `forwardRef(function SquiggleViewer(...) {})` instead of `forwardRef(() => ...)`, which should be enough), but other than that, it's quite convenient.

There's also right click -> "Store as global variable" action on individual props (hard to screenshot).

IIRC, until recently, Next.js obfuscated component names in production builds, but recently they changed it to look the same as in dev.

Reply:
It might not be very useful, but it seems pretty cheap. And the React dev tools are a browser extention - a fair bit more hacky.

This we could give to users for use.